lib/active_model/serializer.rb

```diff
@@ -27,6 +27,20 @@ class Serializer
       )
     /x
 
+    # Hashes contents of file for +_cache_digest+
+    def self.digest_caller_file(caller_line)
+      serializer_file_path = caller_line[CALLER_FILE]
+      serializer_file_contents = IO.read(serializer_file_path)
+      Digest::MD5.hexdigest(serializer_file_contents)
+    rescue TypeError, Errno::ENOENT
+      warn <<-EOF.strip_heredoc
+       Cannot digest non-existent file: '#{caller_line}'.
+       Please set `::_cache_digest` of the serializer
+       if you'd like to cache it.
+       EOF
+      ''.freeze
```

Comment:
:+1: very descriptive error message explains the problem, and says what to do about it! yay
:tada: